README.md

```diff
@@ -6,7 +6,7 @@
 
 Rollup plugin for typescript with compiler errors.
 
-This is a rewrite of original rollup-plugin-typescript, starting and borrowing from [this fork](https://github.com/alexlur/rollup-plugin-typescript).
+This is a rewrite of original [`rollup-plugin-typescript`](https://github.com/rollup/rollup-plugin-typescript)(The library is no longer maintained, and now available at [`@rollup/plugin-typescript`](https://github.com/rollup/plugins/tree/master/packages/typescript)), starting and borrowing from a fork by `alexlur`.
```

Comment:
IMO, if this were to be merged:
- I think there's no need to link to `@rollup/plugin-typescript` since the archived repo does that already and most folks familiar with Rollup would know that too
- I don't think we should remove the link to alexur's fork, even if it doesn't exist anymore. It might be private right now or reuploaded later. Or it could be available on the Wayback Machine etc. So the link is useful even if it currently 404s

It might also be good to link to an _older_ commit of `rollup-plugin-typescript` for posterity. It looked like [`0.8.1`](https://github.com/rollup/rollup-plugin-typescript/tree/v0.8.1) may be where the fork started, or potentially earlier.
Definitely _not_ later though, as the next tag, `1.0.0`, came after this repo's first few commits, e.g. afc02fc2f92567748eef2f06c393d96589202eeb is a good reference

So, with that said, my personal suggestion would be:

```diff
This is a rewrite of the [original `rollup-plugin-typescript`](https://github.com/rollup/rollup-plugin-typescript/tree/v0.8.1), starting and borrowing from [@alexur's fork]((https://github.com/alexlur/rollup-plugin-typescript).
```

This would be simpler to read and maintains some of OP's changes as well

Reply:
Added this in #380